### PR TITLE
Integrate mock events with toast ui for home page

### PR DESCRIPTION
**Pull request by harshith-venkatesh**
### What is the change?
Integration and addition of mock events to check for the requirement of our being met

Provide a small description of what did you change and provide the reference to the issue ticket.
Have integrated and created few set of mock events to our product and updated our home page to show the event page

### Is it bug?
No
- Steps to repro
- Expected
- Actual

### \*Dev Tested?

- [X] Yes
- [ ] No

#### Platforms

- [ X] Web

#### Browsers

- [ X] Chrome
- [ X] Safari
- [ X] Firefox

### Before / After Change Screenshots
Screenshot 2022-12-04 at 9 53 10 AM
e-dd32-4250-bf5f-63bd5506b018.png">

> For visual or interaction changes. Can be video / screenshot.

### CODE REVIEW

**Review by DeRaowl — APPROVED** [review]
Have merge conflict

File: app/components/Calendar.client.tsx [hunk 1]
```diff
@@ -1,9 +1,169 @@
...
+    name: "My Calendar",
+    color: "#ffffff",
+    bgColor: "#9e5fff",
+    dragBgColor: "#9e5fff",
+    borderColor: "#9e5fff"
+  }
+];
+
+const Calendar = () => {
+  const cal = useRef<any>();
```
Comment (isVivek99) [inline app/components/Calendar.client.tsx:61]: can we have a **type** here instead of **any**?

File: app/components/Calendar.client.tsx [hunk 2]
```diff
@@ -1,9 +1,169 @@
...
+
+  const onClickSchedule = useCallback((e: {schedule: ISchedule}) => {
+    const { calendarId, id } = e.schedule;
+    const el = cal.current.calendarInst.getElement(id, calendarId);
+
+    console.log(e, el.getBoundingClientRect());
+  }, []);
+
+  const onBeforeCreateSchedule = useCallback((scheduleData: any) => {
```
Comment (isVivek99) [inline app/components/Calendar.client.tsx:70]: should we create an interface for scheduleData?

File: app/components/Calendar.client.tsx [hunk 3]
```diff
@@ -1,9 +1,169 @@
...
+        class: scheduleData.raw["class"]
+      },
+      state: scheduleData.state
+    };
+
+    cal.current.calendarInst.createSchedules([schedule]);
+  }, []);
+
+  const onBeforeDeleteSchedule = useCallback((res: IEventScheduleObject) => {
+    console.log(res);
```
Comment (isVivek99) [inline app/components/Calendar.client.tsx:92]: can we remove logs?

File: app/components/Calendar.client.tsx [hunk 4]
```diff
@@ -1,9 +1,169 @@
...
+
+    const { id, calendarId } = res.schedule;
+
+    cal.current.calendarInst.deleteSchedule(id, calendarId);
+  }, []);
+
+  const onBeforeUpdateSchedule = useCallback((e : { schedule: ISchedule, changes: any}) => {
```
Comment (isVivek99) [inline app/components/Calendar.client.tsx:99]: should we add an interface for **changes** ?

Comment (harshith-venkatesh) [reply app/components/Calendar.client.tsx:99]: added

File: app/components/Calendar.client.tsx [hunk 5]
```diff
@@ -1,9 +1,169 @@
...
+  function _getFormattedTime(time: DateType) {
+    const date = new Date(time.toString());
+    const h = date.getHours();
+    const m = date.getMinutes();
+
+    return `${h}:${m}`;
+  }
+
+  function _getTimeTemplate(schedule: ISchedule , isAllDay: boolean) {
+    var html = [];
```
Comment (yesyash) [inline app/components/Calendar.client.tsx:120]: can we not use var here please also what would be the type of this var?

File: app/components/Calendar.client.tsx (hunk 4 above)
Comment (harshith-venkatesh) [reply app/components/Calendar.client.tsx:90]: Would be updating the type of changes, when we would be working more on onBeforeUpdateSchedule function

**Review by isVivek99 — APPROVED** [review]

**Pull request merged**